### PR TITLE
Updated i18n settings panel

### LINKED ISSUE #1773: Add enable/disable currency settings under localization
Since the drop-down, just a few currencies are disabled by default, but we actually a support a ton of currencies, so shop operators should be able to configure which ones they want to support.

### LINKED ISSUE #1797: Localization Sidebar Fix
Put all languages is in the card component. 
screen shot 2017-02-07 at 11 23 23 am

### PR DESCRIPTION
**Pull request by mikemurray**
Resolves #1797 
Resolves #1773 

- added new package for i18n settings
- converted to react
- added currency list
- added ability to save open state to SettingsCard to user preferences
- added ability to pluck fields in form component to enable concise
forms

### CODE REVIEW

**Comment by kieckhafer** [comment]
Should we have the all off / all on toggle?

**Comment by kieckhafer** [comment]
Base Currency dropdown should filter our all disabled currencies: 

reaction

File: imports/plugins/core/i18n/client/components/localizationSettings.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                type: "select",
+                options: this.props.currencyOptions
+              },
+              baseUOM: {
+                type: "select",
+                options: this.props.uomOptions
+              },
+              language: {
+                type: "select",
+                options: this.props.currencyOptions
```
Comment (hrath2015) [inline imports/plugins/core/i18n/client/components/localizationSettings.js]: `this.props.currencyOptions`  should be language Options.

Comment (hrath2015) [inline imports/plugins/core/i18n/client/components/localizationSettings.js]: Also when you try to select value(Base Language) from available options, expandable arrows from below 2 cards (Languages and Currency) are visible. See the  attached pic. 
capture

**Comment by kieckhafer** [comment]
@hrath2015 you're right, fixing now

**Pull request merged**